x-pack/plugins/security_solution/cypress/integration/detection_rules/override.spec.ts

```diff
@@ -169,8 +169,8 @@ describe('Detection rules, override', () => {
     cy.get(NUMBER_OF_ALERTS)
       .invoke('text')
       .should('match', /^[1-9].+$/); // Any number of alerts
-    cy.get(ALERT_GRID_CELL).contains('auditbeat');
-    cy.get(ALERT_GRID_CELL).contains('critical');
-    cy.get(ALERT_GRID_CELL).contains('80');
+    cy.get(ALERT_GRID_CELL).contains('siem-kibana');
```

Comment:
Why all these values have been changed? In this scenario we are checking the rule override works correctly. In order to do that, we are creating the following rule:

<img width="445" alt="Screenshot 2022-05-20 at 19 27 42" src="https://user-images.githubusercontent.com/17427073/169581125-ce40914c-7031-4ad4-a1ff-c5dda99bb012.png">

We know that the auditbeat document contains:
* agent.type: auditbeat
* destination.port: 80

As per the rule overrides work, we expect for the alert to contain:
* auditbeat as a rule name
* impact critical
* severity 80

Hence, changing that values changes the purpose of the test.

Reply:
undid these changes after discovering the source of the bug. Thanks!